code/procedures/fix-useless-run-values.sh

```diff
@@ -1,8 +1,7 @@
-# This script has to remove all the run-1 and run-2 parts
-# in all the filenames of DTA.
-
 #!/usr/bin/env bash
 
+# This script has to remove all the run-1 and run-2 parts
+# in all the filenames of DTA.
```

Comment:
I'm currently not clear about why this is needed. However, if we really don't need any run values at all, then there's no need for the dicom2spec rules to produce any in the first place. It's not mandatory for the rules to return any.